### PR TITLE
Display service details

### PR DESCRIPTION
**Pull request by peterschwarz**
Display the service details for a member node:

Screen Shot 2020-06-19 at 1 54 36 PM

This are viewable by clicking on the member node in the table of nodes.

### CODE REVIEW

**Review by RyanLassigBanks — DISMISSED** [review]
Tested locally

**Comment by peterschwarz** [comment]
Had to update this to properly parse services in a circuit vs a proposal

**Review by eloaverona — APPROVED** [review]

File: saplings/circuits/src/data/circuits.js [hunk 1]
```diff
@@ -16,13 +16,39 @@
...
+      try {
+        acc[k] = JSON.parse(coll[k]);
+      } catch (e) {
+        acc[k] = coll[k];
+      }
+      return acc;
+    }, {});
+  }
+
+  if (Array.isArray(coll)) {
```
Comment (eloaverona) [inline saplings/circuits/src/data/circuits.js:25]: I just realized, you should check for the array first. The fist if statement will be true for an array as well. The json for the scabbard arguments look like this for a proposal: 
Screen Shot 2020-06-21 at 3 38 17 PM

**Review by eloaverona — CHANGES_REQUESTED** [review]
I meant to request a change with my [last comment](https://github.com/Cargill/splinter-ui/pull/29#discussion_r443254912).

File: saplings/circuits/src/data/circuits.js (hunk 1 above)
Comment (peterschwarz) [reply saplings/circuits/src/data/circuits.js:25]: Good catch

Comment (peterschwarz) [reply saplings/circuits/src/data/circuits.js:25]: Fixed

**Review by eloaverona — DISMISSED** [review]
Tested

**Review by RyanLassigBanks — DISMISSED** [review]

**Comment by peterschwarz** [comment]
Rebased on master, to pick up some changes.

**Review by eloaverona — APPROVED** [review]
Tested

**Review by RyanLassigBanks — APPROVED** [review]

**Pull request merged**